README.md

```diff
@@ -114,6 +114,49 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+#### Publishing messages with schema
+
+The Ably Kafka Connector supports messages which contain schema information by converting them to JSON before publishing them to Ably.
+For example, if messages on the Kafka topic are serialized using Avro with schemas registered at https://<your-schema-registry-host>, then set the following configuration so that those messages are converted from Avro to JSON:
+
+value.converter=io.confluent.connect.avro.AvroConverter
+value.converter.schema.registry.url=https://<your-schema-registry-host>
+`
```

Comment:
This should be surrounded with three backticks ("```") to make it a code block

Reply:
63ac487f62b51c8b32d382202735ab9e4ed48039

Reply:
Thanks. I'm resolving this conversation on @lmars' behalf as I can see that @ikbalkaya has fixed this.